Titanium.Web.Proxy/Network/Tcp/TcpConnection.cs

```diff
@@ -54,7 +54,6 @@ internal TcpConnection()
         public void Dispose()
         {
             Stream?.Close();
-            Stream?.Dispose();
```

Comment:
The Dispose calls only the Close which calls the Dispose(bool dispoing) with true parameter:
https://referencesource.microsoft.com/#mscorlib/system/io/stream.cs,dc4ffe046b847b84,references

This is why I had to remove the Close method override from the CustomBufferedStream, too: the CustomBufferedStream object was not disposed only the base stream.